index.html

```diff
@@ -24,6 +24,90 @@ <h1>Felipe Haybar</h1>
             <a class="textBoton" href="#">Start collaboration</a>
         </button>
         <div class="imagebox"></div>
-    </section>
+        <section class="projects">
+            <h2>Projects</h2>
+            <img class="imgp" src="images/project1.png" alt="project1" width="320px">
+            <div class="pData">
+                <div></div>
+                <h3>Project name goes here</h3>
+                    <ul class="projectUl">
+                        <li class="projectLi">HTML/CSS</li>
+                        <li class="projectLi">Ruby on Rails</li>
+                        <li class="projectLi">Java script</li>
+                    </ul>
+                <button class="see">
+                    <a class="textBoton" href="#">See this project →</a>
+                </button>
+            </div>
+            <img src="images/project2.png" alt="project2" width="320px">
+            <div class="pData">
+                <h3>Project name goes here</h3>
+                    <ul class="projectUl">
+                        <li class="projectLi">HTML/CSS</li>
+                        <li class="projectLi">Ruby on Rails</li>
+                        <li class="projectLi">Java script</li>
+                    </ul>
+                <button class="see">
+                    <a class="textBoton" href="#">See this project →</a>
+                </button>
+            </div>
+            <img src="images/project3.png" alt="project3" width="320px">
+            <div class="pData">
+                <h3>Project name goes here</h3>
+                    <ul class="projectUl">
+                        <li class="projectLi">HTML/CSS</li>
+                        <li class="projectLi">Ruby on Rails</li>
+                        <li class="projectLi">Java script</li>
+                    </ul>
+                <button class="see">
+                    <a class="textBoton" href="#">See this project →</a>
+                </button>
+            </div>
+            <img src="images/project4.png" alt="project4" width="320px">
+            <div class="pData">
+                <h3>Project name goes here</h3>
+                    <ul class="projectUl">
+                        <li class="projectLi">HTML/CSS</li>
+                        <li class="projectLi">Ruby on Rails</li>
+                        <li class="projectLi">Java script</li>
+                    </ul>
+                <button class="see">
+                    <a class="textBoton" href="#">See this project →</a>
```

Comment:
- [ ] Please add user interaction styling to all the buttons (hover, enable ...) in your project as stated in the requirement. 

![b16](https://user-images.githubusercontent.com/103386183/200062007-e837d3a3-b7f8-4348-84f3-223d57a5ac81.png)

Reply:
Hi Oye,
I had to apply hover config to my buttons, nice suggestion
see you